src/Microsoft.EntityFrameworkCore.AutoHistory/Extensions/DbContextExtensions.cs

```diff
@@ -36,7 +36,7 @@ internal static AutoHistory AutoHistory(this EntityEntry entry, string creator =
         {
             var history = new AutoHistory
             {
-                TableName = entry.Entity.GetType().Name,//entry.Metadata.Relational().TableName,
+                TableName = entry.Entity.GetType().Name,
                 CreatedBy = creator,
```

Comment:
This is a breaking change, so we need to add a named `EntityName` propery maybe better, think so?

Reply:
Yes, you're right, the property TableName in Metadata.Relational() is a better approach, we just have to change the name of the property.